python/lambda-from-container/app.py

```diff
@@ -0,0 +1,92 @@
+import os
+from aws_cdk import (
+    core,
+    aws_lambda,
+    aws_ecr,
+    aws_ecr_assets
+)
+
+
+class LambdaContainerFunctionStack(core.Stack):
+    def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+
+        image_name    = "lambdaContainerFunction"
+        image_version = "latest"
+
+        ##
+        ## If use_pre_existing_image is True
+        ## then use an image that already exists in ECR.
+        ## Otherwise, build a new image
+        ##
+        use_pre_existing_image = False
+
+
+
+        ##
+        ## ECR
+        ##
+        if (use_pre_existing_image):
+
+            ##
+            ## Container was build previously, or elsewhere.
+            ## Use the pre-existing container
+            ##
+            ecr_repository = aws_ecr.Repository.from_repository_attributes(self, 
+                id              = "ECR", 
+                repository_arn  = core.Aws.ACCOUNT_ID + ".dkr.ecr." + core.Aws.REGION + ".amazonaws.com/" + image_name + ":" + image_version, 
```

Comment:
This would be more clear as:
```suggestion
                repository_arn  = '{0}.dkr.ecr.{1}.amazonaws.com/{2}:{3}'.format(core.Aws.ACCOUNT_ID, core.Aws.REGION, image_name, image_version), 
```